pkgs/tools/misc/taoup/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'ansi'
```

Comment:
Please add this to rubyPackages instead.

Reply:
Thanks, added in #138240 - will update this shortly.